schema/kbase/jsonschema/credit_metadata.schema.json

```diff
@@ -203,18 +264,47 @@
             "title": "CreditMetadataEntry",
             "type": "object"
         },
+        "Description": {
+            "additionalProperties": false,
+            "description": "Textual information about the resource being represented.",
+            "properties": {
+                "description_text": {
```

Comment:
slight naming difference just so we don't get into an infinite cycle of `description`s!

Reply:
Makes sense!